### ISSUE TITLE
Project generator should use short property names

### ISSUE THREAD

**Issue by vincent-zurczak** [roboconf/roboconf-platform#817]
**version** instead of **application-version**.
Etc.

**Comment by diarraa** [comment]
Is it just replace LEGACY_PREFIX in https://github.com/roboconf/roboconf-platform/blob/master/core/roboconf-core/src/main/java/net/roboconf/core/model/ApplicationTemplateDescriptor.java  by an empty string ?

**Comment by vincent-zurczak** [comment]
No. This is about updating https://github.com/roboconf/roboconf-platform/blob/master/miscellaneous/roboconf-tooling-core/src/main/resources/application-skeleton.props and the associated bundle.

**Comment by diarraa** [comment]
Ok, in this case I think LEGACY_PREFIX = "application-" is useless. Because "application-version" is replaced by "version".

**Comment by vincent-zurczak** [comment]
No, we want to keep backward compatibility.
See #803 :wink:

**Comment by diarraa** [comment]
Ok.
